locale/es/LC_MESSAGES/django.po

```diff
@@ -73,153 +70,152 @@ msgstr "Cargue un archivo"
 msgid "Provide at least one extension URL"
 msgstr "Indique al menos una URL de extensión"
 
-#: default/templates/default/base-uploader.html:15
-#: default/templates/default/to-json.html:13
+#: default/google_drive.py:30
+msgid ""
+"There was an issue when trying to upload the file to Google Drive, please "
+"try again later."
+msgstr "Hubo un error tratando de subir el archivo a Google Drive, favor "
+"intente de nuevo más tarde."
+
+#: default/google_drive.py:31
+msgid ""
+"There was an authorization issue when saving the file to Google Drive, "
+"please try again."
+msgstr "Se encontró un problema de autorización al guardar el archivo en Google Drive, "
+"favor intente de nuevo más tarde"
+
+#: default/templates/default/base-tool.html:14
+msgid "more"
+msgstr "más"
+
+#: default/templates/default/base-tool.html:15
+msgid "less"
+msgstr "menos"
+
+#: default/templates/default/base-tool.html:29
+#: default/templates/default/mapping_sheet.html:43
 #: default/templates/default/to-spreadsheet.html:13
 msgid "Success!"
 msgstr "Éxito!"
 
-#: default/templates/default/base-uploader.html:16
-#: default/templates/default/to-json.html:17
-#: default/templates/default/to-spreadsheet.html:17
-#: default/templates/default/to-spreadsheet.html:21
+#: default/templates/default/base-tool.html:32
+#: default/templates/default/mapping_sheet.html:46
+#: default/templates/default/to-spreadsheet.html:18
+#: default/templates/default/to-spreadsheet.html:31
 msgid "Download"
 msgstr "Descargar"
 
-#: default/templates/default/base-uploader.html:22
+#: default/templates/default/base-tool.html:36
+#: default/templates/default/mapping_sheet.html:50
+#: default/templates/default/to-spreadsheet.html:22
+#: default/templates/default/to-spreadsheet.html:35
+msgid "Save to Google Drive"
+msgstr "Guardar en Google Drive"
+
+#: default/templates/default/base-tool.html:39
+#: default/templates/default/mapping_sheet.html:53
+#: default/templates/default/to-spreadsheet.html:25
+#: default/templates/default/to-spreadsheet.html:38
+msgid "File saved on Google Drive"
```

Comment:
It's better to say "in"?

Reply:
La verdad no estaba segura, así que le pregunté a Teté y me dijo que es "on".